hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/CapacitySchedulerQueueManager.java

```diff
@@ -239,46 +237,47 @@ static CSQueue parseQueue(
       CSQueue parent, String queueName, CSQueueStore newQueues, CSQueueStore oldQueues,
       QueueHook hook) throws IOException {
     CSQueue queue;
-    String fullQueueName = (parent == null) ? queueName :
-        (QueuePath.createFromQueues(parent.getQueuePath(), queueName).getFullPath());
-    String[] staticChildQueueNames = conf.getQueues(fullQueueName);
-    List<String> childQueueNames = staticChildQueueNames != null ?
-        Arrays.asList(staticChildQueueNames) : Collections.emptyList();
-    CSQueue oldQueue = oldQueues.get(fullQueueName);
-
-    boolean isReservableQueue = conf.isReservable(fullQueueName);
-    boolean isAutoCreateEnabled = conf.isAutoCreateChildQueueEnabled(fullQueueName);
+    QueuePath fullQueuePath = (parent == null) ? new QueuePath(queueName) :
```

Comment:
Nit: fullQueuePath is a bit redundant as far as naming goes, as by definition all queuepath objects contain the full queue path. Something like queueToParse would be a bit more descriptive, but retriggering the build because of a new commit only for this is probably not worth it, this easily can be done later.